Makefile

```
@@ -1,82 +1,43 @@
-	@mkdir -p de1soc
-	$(CC) --sysroot=$(SYSROOT) $(CFLAGS_DE1SOC) -static $^ -lpthread -o $@
-
-# Clean build files
+TARGET ?= linux
+
+# Default target
+ifeq ($(MAKECMDGOALS),linux)
+  include config/Makefile.linux
+endif
+
+.PHONY: all linux arm_linux clean
```

checkmake (0.2.2)

[warning] Missing required phony target "test"

(minphony)